tests/system/providers/google/bigquery/example_bigquery_sensors.py

```diff
@@ -23,6 +23,7 @@
 from datetime import datetime
 
 from airflow import models
+from airflow.models.baseoperator import BaseOperator
```

Comment:
Why changing system tests is part of this PR?

Reply:
this came as part of the semi-rndom "mypy" failures. Mypy detected slightly different errors when there was a different set of files - so I had some 20 more mypy failures with this change. 

 I thought it has been merged separately but I will definitely check it (I will myself do a thorough review of it before I merge :).